packages/xchain-bitcoincash/src/utils.ts

```diff
@@ -192,7 +192,7 @@ export const parseTransaction = (tx: Transaction): Tx => {
       .map(
         (input) =>
           ({
-            from: input.address,
+            from: input.address ? stripPrefix(input.address) : input.address,
```

Comment:
Sounds confusing ^. Add a comment to understand `input.address` will always be available, because `input` with available `address` will use here only.